test/index.test.js

```diff
@@ -0,0 +1,31 @@
+const { describe, before, beforeEach, after, it } = require('mocha')
+const { expect } = require('chai')
+const express = require('express')
+const axios = require('axios')
+const createApp = require('../create-app')
+
+describe('app', () => {
+
+  let _repo
+  let server
+
+  before(done => {
+    _repo = { name: 'continuous-delivery', description: 'A practice repository for testing and deployment' }
+    server = createApp()
+      .get('/', (req, res) => res.json(_repo))
```

Comment:
The server should already know how to respond. Don't rewrite the server code in the tests.